index.html

```diff
@@ -44,5 +51,141 @@ <h3>I’am a Software Developer</h3>
+            <li>Ruby</li>
+          </ul>
+          <button>See Project</button>
+        </div>
+        <div class="art">
+          <h1>Profesional Art Printing Data</h1>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard
+          </p>
+          <ul>
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button>See Project</button>
+        </div>
+        <div class="art">
+          <h1>Profesional Art Printing Data</h1>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard
+          </p>
+          <ul>
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button>See Project</button>
+        </div>
+        <div class="art">
+          <h1>Profesional Art Printing Data</h1>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard
+          </p>
+          <ul>
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button>See Project</button>
+        </div>
+        <div class="art">
+          <h1>Profesional Art Printing Data</h1>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard
+          </p>
+          <ul>
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button>See Project</button>
+        </div>
+      </div>
+    </section>
+
+    <section class="aboutMe">
+      <h1>About me</h1>
+      <p>
+        Hello I’m a software developer! I can help you build a product , feature
+        or website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <button>Get My Resume</button>
+      <img class="vector" src="./images/Devider.png" alt="" />
+      <ul>
+        <li class="first">
+          <img src="./images/Rectangle 55.png" alt="" /> Language
+        </li>
+        <li>Javascript</li>
+        <li>Ruby</li>
+        <li>Html</li>
+        <li>Css</li>
+        <li class="first">
+          <img src="./images/Rectangle 56.png" alt="" /> Frameworks
+        </li>
+        <li>Bootstrap</li>
+        <li>Ruby on Rails</li>
+        <li>RSPec</li>
+        <li>CapyBara</li>
+        <li>Selenium</li>
+        <li class="first">
+          <img src="./images/Rectangle 57.png" alt="" /> Skills
+        </li>
+        <li>Codekit</li>
+        <li>Github</li>
+        <li>Codepen</li>
+        <li>Gitlab</li>
+        <li>Terminal</li>
+      </ul>
```

Comment:
Kindly wrap the `Languages`, `Frameworks` and `Skills` inside `ul` and each set of technologies inside another `ul` tag.

Reply:
I couldn't understand. Please can you explain this a bit more? @gemmen29

Reply:
It will be like that
```
<ul>
     <li>
           Languages
           <ul>
                 technologies list.........................
           </ul>
     </li>
     <li>
           Frameworks
            <ul>
                 technologies list.........................
           </ul>
     </li>
     <li>
            Skills
            <ul>
                 technologies list.........................
           </ul>
     </li>
</ul>
```